.github/workflows/ci.yml

```diff
@@ -0,0 +1,91 @@
+---
+name: CI
+'on':
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+env:
+  MAKEFLAGS: -j2
```

Comment:
This was ported as-is, but maybe there's a better way to find a suitable value? e.g. `export MAKEFLAGS="-j$(getconf _NPROCESSORS_ONLN)"` from within the `run` below (?)

Reply:
@jadeallenx, any opinion on this? It would just affect `kerl`'s workflows, but faster is better than slower, probably 😄

Reply:
I'ma apply my suggestion, then. Or did you mean "good to me (as-is)"?

Reply:
I'm running CI in my fork, again, to see how much of a time reduction we get. Before, it took from [(min.) 7"55'](https://github.com/paulo-ferraz-oliveira/kerl/actions/runs/3466045623/jobs/5789461325#step:6:1) to [(max.) 20"47'](https://github.com/paulo-ferraz-oliveira/kerl/actions/runs/3466045623/jobs/5789461664#step:6:1).